app/controllers/boxes_controller.rb

```diff
@@ -91,6 +92,21 @@ def bulk_destroy
     redirect_to boxes_path, notice: "Boxes were successfully deleted."
   end
 
+  def unblind
+    return unless authorize_resource(@box, UPDATE_BOX)
```

Comment:
Sadly, this won't do. Once the box is added to a transfer we won't show the blind/unblind button, but once the transfer is confirmed, the receiver institution will own the box, and thus be authorized to blind/unblind by forging the request.

A basic `return head :forbidden if @box.transferred?` should work for now.